docs/qprofiler2/index.md

```diff
@@ -308,7 +306,7 @@ In above bam summary output example, this metrics lists the summed counts from a
 
 
 ### BAM metrics output
-The summary of BAM record fileds is outputed to "bamMetrics" section. There are too much information, here we only lists "readCount" description under each "sequneceMetrics" node.
+This <bamMetrics> element lists static data based on BAM record fields order, including all mandatory fields and some optional fields in the SAM format. In each BAM record field, qprofiler2 takes different read counts. eg. it only counts mapped records to summary the POS field, but counts all records to summary the FLAG field. The details lists below:
```

Comment:
"For example" rather than "eg." when starting a sentence.
last 2 sentences need some work